### ISSUE TITLE
SyntaxError: Cannot use import statement outside a module

### ISSUE THREAD

**Issue by apustula** [fmkra/next-gallery#1]
Hello,
I was trying to use your library in Nextjs, Ts app and get an error.
Using template inside `/pages` directory:
``` js
import React from "react";
import { Gallery } from "next-gallery";

const images = [
  { src: "https://picsum.photos/id/1018/1920/1080/", aspect_ratio: 16 / 9 },
  { src: "https://picsum.photos/id/1015/1920/1080/", aspect_ratio: 16 / 9 },
];
const widths = [500, 1000, 1600];
const ratios = [2.2, 4, 6, 8];

const Portfolio = () => {
  return (
    <div className="">
      <Gallery {...{ images, widths, ratios }} />
    </div>
  );
};

export default Portfolio;
```

Leads to an error:
```
- error .../Projects/my-app/node_modules/next-gallery/dist/index.js:1
import { useState, useEffect, useMemo } from 'react';
^^^^^^

SyntaxError: Cannot use import statement outside a module
```
Dependencies version:
    "next": "^13.4",
    "next-gallery": "^1.2.1",
    "react": "^18.2.0",
    "react-dom": "^18.2.0",
    "typescript": "^4.7.4"

**Comment by fmkra** [comment]
I will be working on this issue. However a temporary workaround would be to add
```ts
transpilePackages: ['next-gallery']
```
to your `next.config.js`.

**Comment by apustula** [comment]
Thanks a lot

**Comment by fmkra** [comment]
This is now fixed in version 2.0.0
